src/signals/incident-management/containers/IncidentSplitContainer/components/IncidentSplitFormIncident/IncidentSplitFormIncident.tsx

```diff
@@ -130,6 +130,9 @@ const IncidentSplitFormIncident: FC<IncidentSplitFormIncidentProps> = ({
               render={({ field, fieldState: { error } }) => (
                 <StyledAddNote
                   {...field}
+                  description="Zorg ervoor dat hier duidelijk staat wat het specifieke probleem is
+                  dat de afdeling moet behandelen. Verwijder uit de meldtekst wat niet
+                  van belang is. En verduidelijk het verzoek waar nodig is."
```

Comment:
`En verduidelijk het verzoek waar nodig.` or `En verduidelijk het verzoek waar dat nodig is.`

https://taaladvies.net/wanneer-nodig-of-wanneer-dat-nodig-is/

Reply:
Sharp